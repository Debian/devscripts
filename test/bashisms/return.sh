#!/bin/sh

foo() {
    return -- 1 # BASHISM
}

bar () {
    return 256 # BASHISM
}

moo () {
    return -1 # BASHISM
}
