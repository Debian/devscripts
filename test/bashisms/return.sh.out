possible bashism in bashisms/return.sh line 4 ('exit --' should be 'exit' (idem for return)):
    return -- 1 # BASHISM
possible bashism in bashisms/return.sh line 8 (exit|return status code greater than 255):
    return 256 # BASHISM
possible bashism in bashisms/return.sh line 12 (exit|return with negative status code):
    return -1 # BASHISM
